src/test/java/com/cegeka/academy/repository/EventRepositoryTest.java

```diff
@@ -73,14 +73,14 @@ public void testFindAllByIsPublicIsTrue() {
         for (int i = 0; i < 5; i++) {
             Event event = TestsRepositoryUtil.createEvent("Ana are mere!", "KFC Krushers Party", true, address, user, categories, "https://scontent.fotp3-2.fna.fbcdn.net/v/t1.0-9/67786277_2592710307438854_5055220041180512256");
             if (i % 2 == 0) {
```

Comment:
Scapa de `if`.

Reply:
Am pus direct conditia in setter. E bine?